internal/util/streamrpc/streamer.go

```diff
@@ -102,25 +102,66 @@ func mergeCostAggregation(a *internalpb.CostAggregation, b *internalpb.CostAggre
 	return &internalpb.CostAggregation{
 		ResponseTime:         a.GetResponseTime() + b.GetResponseTime(),
 		ServiceTime:          a.GetServiceTime() + b.GetServiceTime(),
-		TotalNQ:              a.GetTotalNQ() + b.GetTotalNQ(),
+		TotalNQ:              a.GetTotalNQ(),
```

Comment:
Why is this changed?

Reply:
The old TotalNq is wrong but not used before. Nq should be same between results in same query.